### PR TITLE
Cherry-picked locale updates: it_IT and fr_CA

### PR DESCRIPTION
**Pull request by mtub**
We continue to update ojs-stable-2_4_8 preparing for OJS 2.4.8-1 release...

Original PRs (ojs-dev-2_4): #895, #907

### CODE REVIEW

**Comment by mtub** [comment]
@asmecher , can you take a look at this and merge it?

File: plugins/reports/counter/locale/it_IT/ar1.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/reports/counter/locale/en_US/ar1.xml
```
Comment (asmecher) [inline plugins/reports/counter/locale/it_IT/ar1.xml]: I'm not sure what this file is -- maybe a work in progress that got accidentally added?

File: plugins/reports/counter/locale/it_IT/jr1.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/reports/counter/locale/en_US/jr1.xml
```
Comment (asmecher) [inline plugins/reports/counter/locale/it_IT/jr1.xml]: I'm not sure what this file is -- maybe a work in progress that got accidentally added?

**Comment by mtub** [comment]
Those two files are the counter report template files that are included in en_US and other locales. Translator didn't update the en_US part of the path in the XML file header, but other than that this should be okay.

**Comment by mtub** [comment]
(I am not saying that these files should be in the release - I couldn't find any process making use of them. But the files have not been introduced by these commits, but had been included in en_US from where translators have picked them up and worked with them.)

**Comment by asmecher** [comment]
OK, thanks -- merged!

**Comment by mtub** [comment]
So I'd rather merge this and postpone deleting the files in all locales after verifying they are not needed...

**Comment by mtub** [comment]
Well thanks :)

**Comment by mtub** [comment]
@asmecher , I am not sure if you've read my latest comments before merging - maybe someone should check if these files are really needed…

**Comment by asmecher** [comment]
Yes, I double-checked and they're indeed used -- it's just a little unconventional so I didn't recognize what was happening at first.

**Pull request merged**